components/Dashboard/index.tsx

```diff
@@ -38,12 +39,12 @@ const Dashboard: FunctionComponent<{}> = () => {
               className="pv1 ph3 br1 f7 ml1 c-find-by-skills-button"
               type="button"
             >
-              Search
+              <Search className="w1"/>
             </Button>
           </div>
         </div>
       </section>
-      <section className="pv4 flex justfy-center">
+      <section className="c-ReccommendedMentorshipCardWrapper pv4 flex-m justify-between-m flex-l justify-between-l">
```

Comment:
`c-ReccommendedMentorshipCardWrapper ` can be renamed to just `c-mentorship-section`

Reply:
Okay will work on the rename

Reply:
I just remove the custom class name. I noticed I didn't make use of it any longer